### DISCUSSION TITLE
Unifi Access Point

### DISCUSSION THREAD

**Discussion by NewSmokie** [hassio-addons/addon-unifi#472] [Q&A]
Guten Tag zusammen ich bin neu hier. Ich baue grade meine Home Assistant auf und habe das Problem das ich mein Unifi nicht ins home Assistent integriert bekomme.
Habe jedes Video geschaut und die Beschreibungen gelesen leider ohne Erfolg.
Ich bin soweit gekommendas ich die addon installiert habe und gestartet habe danach. Komme ich nicht weiter 

Danke für eure Hilfe

**Comment by renewoensdregt** [comment]
Please for issues around usage and not bugs use the home assistant forum and in English please.
________________________________
From: NewSmokie ***@***.***>
Sent: Sunday, December 24, 2023 4:31:03 PM
To: hassio-addons/addon-unifi ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [hassio-addons/addon-unifi] Unifi Access Point (Discussion #472)


Guten Tag zusammen ich bin neu hier. Ich baue grade meine Home Assistant auf und habe das Problem das ich mein Unifi nicht ins home Assistent integriert bekomme.
Habe jedes Video geschaut und die Beschreibungen gelesen leider ohne Erfolg.
Ich bin soweit gekommendas ich die addon installiert habe und gestartet habe danach. Komme ich nicht weiter

Danke für eure Hilfe
